### PR TITLE
Add aviator config

### PR DESCRIPTION
**Pull request by samfishman**

### CODE REVIEW

File: aviator.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+gearman-admin-env:
+  setups:
+    default:
+      publish:
+        google-id: ""
+        google-secret: ""        
+        allowable-domains: '["localhost"]'
+        node-env: "development"
+        db-type: "sqlite"
```
Comment (MaerF0x0) [inline aviator.yml]: i wonder about configuring it to spin up a db like we do with redis/mongo and clever-ops?

Comment (samfishman) [inline aviator.yml]: We could definitely do that but it seems easier to just use sqlite for development. What are the benefits of spinning up a Postgres db?

Comment (MaerF0x0) [inline aviator.yml]: More like prod. No worries.

**Comment by MaerF0x0** [comment]
LGTM

[LGTM](http://lgtm.in/i/EOFu15qaq)

**Pull request merged**